pallets/pools/src/lib.rs

```diff
@@ -429,7 +433,12 @@ pub mod pallet {
 		_,
 		Blake2_128Concat,
 		T::PoolId,
-		ScheduledUpdateDetails<T::InterestRate, T::MaxTokenNameLength, T::MaxTokenSymbolLength>,
+		ScheduledUpdateDetails<
+			T::InterestRate,
+			T::MaxTokenNameLength,
+			T::MaxTokenSymbolLength,
+			T::MaxTranches,
```

Comment:
I've used for the bound the `MaxTranches` which seems to be the right value. Can Anyone involved in `pallet-pools` confirm this?

Reply:
This is correct.